extensions/big-data-cluster/src/bigDataCluster/utils.ts

```diff
@@ -273,3 +271,34 @@ export function getControllerEndpoint(serverInfo: azdata.ServerInfo): string | u
 export function getBdcStatusErrorMessage(error: Error): string {
 	return localize('endpointsError', "Unexpected error retrieving BDC Endpoints: {0}", error.message);
 }
+
+export const bdcConfigSectionName = 'bigDataCluster';
+export const ignoreSslConfigName = 'ignoreSslVerification';
+
+/**
+ * Retrieves the current setting for whether to ignore SSL verification errors
+ */
+export function getIgnoreSslVerificationConfigSetting(): boolean {
+	return _ignoreSslVerification;
+}
+
+let _ignoreSslVerification = false;
+
+vscode.workspace.onDidChangeConfiguration((e: vscode.ConfigurationChangeEvent) => {
+	if (e.affectsConfiguration(`${bdcConfigSectionName}.${ignoreSslConfigName}`)) {
+		updateIgnoreSslConfigSetting();
+	}
+});
+
+function updateIgnoreSslConfigSetting(): void {
+	try {
+		const config = vscode.workspace.getConfiguration(bdcConfigSectionName);
+		_ignoreSslVerification = config.get<boolean>(ignoreSslConfigName) || false;
+	} catch (error) {
+		console.error(`Unexpected error retrieving ${bdcConfigSectionName}.${ignoreSslConfigName} setting : ${error}`);
+	}
+}
+
+updateIgnoreSslConfigSetting();
```

Comment:
I don't really like this implementation, can we just always query the real value in the `getIgnoreSslVerificationConfigSetting` function instead?